Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -543,8 +542,11 @@ private void ResettingCleanup()
             _playerGameStatuses.Clear();
             foreach (var session in _playerManager.Sessions)
             {
-                _playerGameStatuses[session.UserId] = LobbyEnabled ?  PlayerGameStatus.NotReadyToPlay : PlayerGameStatus.ReadyToPlay;
+                _playerGameStatuses[session.UserId] = LobbyEnabled ? PlayerGameStatus.NotReadyToPlay : PlayerGameStatus.ReadyToPlay;
             }
+
+            // Put a bangin' donk on it.
+            _audio.PlayGlobal(new SoundCollectionSpecifier("RoundEnd"), Filter.Broadcast(), true);
         }
```

Comment:
This should get the soundpathspecifier so it's at least consistent across clients and not random.

Reply:
I've refactored to specifically feed in a specific file resource string - let me know if there's some specific code you'd prefer.